buildbot/configure.py

```diff
@@ -139,9 +128,4 @@ def main():
 
     print("args:{}".format(args))
 
-    return do_configure(args)
-
-if __name__ == "__main__":
```

Comment:
With the removal of this, can I still call the script standalone to test changes locally?

Reply:
Yes. I use it locally and it works for me.

Reply:
CI tells me that I'm doing something wrong. I'll rollback this part.